src/GlobalHeader/index.js

```diff
@@ -134,280 +142,267 @@ const GlobalHeader = ({ globalNav, pages, docs, location, ...props }) => {
-                        <div
-                          className={classNames(
-                            'spectrum-Dropdown',
-                            'spectrum-Dropdown--quiet',
-                            {
-                              'is-open': isPrimary
-                                ? openPrimaryMenu
-                                : openSecondaryMenu
-                            }
-                          )}
-                        >
-                          <button
-                            className={classNames(
-                              'spectrum-FieldButton',
-                              'spectrum-FieldButton--quiet',
-                              'spectrum-Dropdown-trigger',
-                              {
-                                'is-selected': isPrimary
-                                  ? openPrimaryMenu
-                                  : openSecondaryMenu
-                              }
-                            )}
-                            aria-haspopup='listbox'
-                            aria-expanded={
-                              isPrimary ? openPrimaryMenu : openSecondaryMenu
-                            }
-                            aria-controls={id}
-                            onClick={(event) => {
-                              event.stopPropagation()
-                              event.nativeEvent.stopImmediatePropagation()
+                          : ''}
+                      `}
+                    >
+                      <PickerButton
+                        isQuiet
+                        isOpen={isPrimary ? openPrimaryMenu : openSecondaryMenu}
+                        ariaControls={id}
+                        onClick={(event) => {
+                          event.stopPropagation()
+                          event.nativeEvent.stopImmediatePropagation()
 
-                              if (isPrimary) {
-                                setOpenPrimaryMenu(!openPrimaryMenu)
-                                setOpenSecondaryMenu(false)
-                              } else {
-                                setOpenSecondaryMenu(!openSecondaryMenu)
-                                setOpenPrimaryMenu(false)
-                              }
-                            }}
-                          >
-                            <span className='spectrum-Dropdown-label'>
-                              {menu.title}
-                            </span>
-                            <ChevronDown className='spectrum-Dropdown-icon' />
-                          </button>
-                        </div>
-                        <div
-                          ref={isPrimary ? primaryPopover : secondaryPopover}
-                          id={id}
-                          aria-hidden={
-                            isPrimary ? !openPrimaryMenu : !openSecondaryMenu
+                          if (isPrimary) {
+                            setOpenPrimaryMenu(!openPrimaryMenu)
```

Comment:
Should be:

`setOpenPrimaryMenu(openPrimaryMenu => !openPrimaryMenu)`

https://dev.to/alexkhismatulin/update-boolean-state-right-with-react-hooks-3k2i